docs/guides/docs/intro-to-sway/typescript-sdk.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: Typescript Frontend
+category: Intro to Sway
+parent:
+  label: Guides
+  link: /guides
+---
+
+## Building the Frontend
+
+### Setup
+
+Initialize a new React app with TypeScript in the same parent folder as your contract using the command below.
+
+```shell
+npx create-react-app frontend --template typescript
```

Comment:
I understand that the example was already generated using CRA. However, should we move forward into using vite?

Reply:
I think this is fine for now